app/controllers/nonsipp/bondsdisposal/RemoveBondsDisposalController.scala

```diff
@@ -58,6 +58,9 @@ class RemoveBondsDisposalController @Inject()(
     identifyAndRequireData(srn) { implicit request =>
       (
         for {
+          _ <- request.userAnswers
```

Comment:
Would it be possible to test this? E.g. https://github.com/hmrc/pension-scheme-return-frontend/blob/b293d005c8d95ff8e1abfb29168df3269f8c80a8/test/controllers/nonsipp/bonds/RemoveBondsControllerSpec.scala#L63